tck-download/jakarta-websocket-tck/pom.xml

```diff
@@ -52,26 +53,24 @@
                 </executions>
                 <configuration>
                     <url>${tck.test.websocket.url}</url>
-                    <unpack>false</unpack>
+                    <unpack>true</unpack>
                     <outputDirectory>${project.build.directory}</outputDirectory>
                 </configuration>
             </plugin>
             <plugin>
-                <artifactId>maven-install-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.3.0</version>
                 <executions>
                     <execution>
-                        <id>install-servlet-tck</id>
-                        <phase>process-resources</phase>
+                        <id>installing artifacts</id>
+                        <phase>install</phase>
                         <goals>
-                            <goal>install-file</goal>
+                            <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <file>${project.build.directory}/${tck.test.websocket.file}</file>
-                            <groupId>${project.groupId}</groupId>
-                            <artifactId>${project.artifactId}</artifactId>
-                            <version>${project.version}</version>
-                            <packaging>zip</packaging>
-                            <generatePom>true</generatePom>
+                            <executable>${project.build.directory}${file.separator}websocket-tck${file.separator}artifacts${file.separator}artifact-install.sh</executable>
```

Comment:
Unnecessarily restrictive towards Windows.
Replace with OS-agnostic `install-file` directives please.
See [here](https://github.com/payara/jakartaee-10-tck-runners/blob/EE11/tck-download/jakarta-expression-language-tck/pom.xml#L95) for an example

Reply:
already done!